### DISCUSSION TITLE
How do you disable the "fullbright"

### DISCUSSION THREAD

**Discussion by dutyunogasai** [AnOpenSauceDev/Methane-mod#71] [Q&A]
This may be a dumb question but I am trying to figure out how to disable the "Fullbright" or gamma or whatever that methane puts on your game and I saw on the modrinth page that you can disable it with mod menu, but when I checked out all the settings in mod menu I can't find any setting to turn this lighting off, is there another way you're supposed to do it? Also, idk if this is important or not but I am playing on a server.

**Comment by AnOpenSauceDev** [comment]
Methane should have a settings button when you click on the mod menu entry for the mod.

**Reply by dutyunogasai** [reply]
yes it does, but I've tried turning all of these settings on and off, but none of them seem to affect this fullbright effect

**Reply by AnOpenSauceDev** [reply]
make sure to set "start with Methane OFF" to yes, "use old lighting engine" to yes, and if that doesn't work, just use the keybind with the old lighting engine (in controls).

**Reply by dutyunogasai** [reply]
I am using version 2.5 and I dont see a "use old lighting engine" setting
![2023-11-12_17 14 27](https://github.com/AnOpenSauceDev/Methane-mod/assets/145818855/b2eb207f-6256-43e0-a449-8b9bd6cbb6cb)

**Reply by AnOpenSauceDev** [reply]
That shouldn't matter, as you are on 2.5 (which uses the old engine)
